HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
```

Comment:
Add attribute for table name `[Table()]` (Or something like that I forget the exact attribute name)